src/utils/TodoContext.tsx

```diff
@@ -0,0 +1,205 @@
+import React, {
+  createContext,
+  useContext,
+  useState,
+  FC,
+  useEffect,
+} from 'react';
+import { Todo } from '../types/Todo';
+import { TodosContextType } from '../types/TodosContextType';
+import { Filter } from '../types/Filter';
+import {
+  deleteTodoFromServer,
+  getTodos,
+  renameTodoOnServer,
+  sendTodoToServer,
+  toggleStatusTodo,
+} from '../api/todos';
+import { ErrorMessage } from '../types/ErrorMessage';
+
+const initialTodos: Todo[] = [];
```

Comment:
I'm not sure that we really need this variable

Reply:
> I'm not sure that we really need this variable

Not fixed without any reply, the same as a few comments below 🤷